### PR TITLE
Alert avatar P0

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Have added the alert and avatar components for the website. Please review

### CODE REVIEW

File: component.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      <h3 class="avatar-heading">Avatar</h3>
+      <p class="component-navbar-item">
+        In computing, an avatar (also known as a profile picture or userpic) is
+        a graphical representation of a user or the user's character or persona.
+      </p>
+      <div class="avatar-items">
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-xl" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-l" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-md" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-s" />
```
Comment (anshulraheja) [inline component.html]: 'l' and 's' in the class name doesn't convey its use instead use 'avatar-lg' and  'avatar-sm'

File: style/nav.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  align-items: center;
+  
+}
+.left-main-section-container {
+  flex-basis: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-top: 80px;
```
Comment (Ramandeep9898) [inline style/nav.css]: u can make diff. classes for flex... so it will help u not to repeat code.

File: style/Avatar.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.avatar-heading{
+  margin-top:80px ;
+    padding: 0px 20px 20px 20px;
+    font-size: 30px;
+    color:var(--background-color);
+}
+
+.component-navbar-item{
```
Comment (anshulraheja) [inline style/Avatar.css:19]: use rem for large values

**Review by anshulraheja — APPROVED** [review]
clean UI

File: style/nav.css (hunk 2 above)
Comment (yashodaAgarwal) [reply style/nav.css]: Sure I will do that.Thankyou

File: component.html (hunk 1 above)
Comment (yashodaAgarwal) [reply component.html]: Okay sure I will change that.Thankyou

File: component.html [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+          overflow: hidden;
+        "
+        sandbox="allow-scripts allow-same-origin"
+      >
+      </iframe>
+    </section>
+
+    <!-- ---- alert ---- -->
+
+    <section class="main-title" id="alerts">
```
Comment (ashishkatiyar2404) [inline component.html:116]: classes will be enough to use.
use ID when you want to override some properties

File: index.html [hunk 5]
```diff
@@ -1,12 +1,87 @@
...
+              >Installation</a
+            >
+          </div>
+        </div>
+        <div class="right-main-section-container">
+          <img
+            class="image"
+            src="Asset/background.png"
+            height="500px"
+            width="500px"
```
Comment (ashishkatiyar2404) [inline index.html]: avoid inline styling

**Review by ashishkatiyar2404 — CHANGES_REQUESTED** [review]

File: style/Alerts.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+.alert-items{
+    padding:10px 10px 10px 10px;
+    margin:20px 0px 20px 0px;
+    border-radius: 10px;
+    border: 0.5px solid rgba(128, 128, 128, 0.363);
+}
+.alert-icon{
+    padding:0px 10px 0px 10px
+}
```
Comment (ashishkatiyar2404) [inline style/Alerts.css:30]: use root variables for colors

File: component.html (hunk 4 above)
Comment (yashodaAgarwal) [reply component.html:116]: I have used this id to connect it to my sidebar

**Review by ashishkatiyar2404 — CHANGES_REQUESTED** [review]

File: style/nav.css [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+.navbar-list a {
+  text-decoration: none;
+}
+.Logo {
+  padding: 20px 20px 20px 90px;
+  font-size: 30px;
+  text-decoration: none;
+  color: var(--white-color);
```
Comment (ashishkatiyar2404) [inline style/nav.css:33]: in nav.css here u used root variables thats good

**Review by ashishkatiyar2404 — APPROVED** [review]

File: component.html (hunk 1 above)
Comment (yashodaAgarwal) [reply component.html]: I have made the necessary changes. Can you please check on that

File: style/Alerts.css (hunk 6 above)
Comment (yashodaAgarwal) [reply style/Alerts.css:30]: I have used

File: index.html (hunk 5 above)
Comment (yashodaAgarwal) [reply index.html]: Okay

File: style/Avatar.css (hunk 3 above)
Comment (yashodaAgarwal) [reply style/Avatar.css:19]: Sure

File: component.html (hunk 4 above)
Comment (ashishkatiyar2404) [reply component.html:116]: ok. cool💯

File: style/Alerts.css (hunk 6 above)
Comment (ashishkatiyar2404) [reply style/Alerts.css:30]: sorry I think i selected the wrong lines😅

**Review by ashishkatiyar2404 — APPROVED** [review]

**Review by ashishkatiyar2404 — APPROVED** [review]

**Review by ashishkatiyar2404 — APPROVED** [review]

File: style/nav.css (hunk 2 above)
Comment (Ramandeep9898) [reply style/nav.css]: 👍👍👍

**Pull request merged**